src/Microsoft.AspNet.Mvc.Localization/project.json

```diff
@@ -16,9 +16,13 @@
     "dnx451": {},
     "dnxcore50": {}
   },
-  "exclude": [
-    "wwwroot",
-    "node_modules",
-    "bower_components"
-  ]
+    "exclude": [
+        "wwwroot",
+        "node_modules",
+        "bower_components"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aspnet/mvc"
+    }
```

Comment:
@dougbu updated (I used the default settings of my VS RTM which is set to use 4 spaces per tab)